pkg/common/connectionmanager/connectionmanager.go

```diff
@@ -268,6 +274,9 @@ func (cm *ConnectionManager) WhichVCandDCByNodeId(nodeID string, searchBy FindVM
 		return found
 	}
 
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
It's from the original code, but don't think there's a need for using `WithCancel` here, just `ctx` as-is should be fine.

Reply:
Will remove